packages/lib/models/settings/builtInMetadata.ts

```diff
@@ -1566,6 +1566,17 @@ const builtInMetadata = (Setting: typeof SettingType) => {
 		// 	storage: SettingStorage.File,
 		// },
 
+		'featureFlag.useBetaEncryptionMethod': {
+			value: false,
+			type: SettingItemType.Bool,
+			public: true,
+			storage: SettingStorage.File,
+			label: () => 'Use beta encryption',
+			description: () => 'Set beta encryption methods as the default methods. This applies to all clients and takes effect after restarting the app',
```

Comment:
**🟢 UI**: After being released, I think this setting is most likely to be changed by technical users. If so, it could make sense to move it under "Advanced". 

```suggestion
			description: () => 'Set beta encryption methods as the default methods. This applies to all clients and takes effect after restarting the app.',
			advanced: true,
```

Reply:
Finished in commit bb69f50bfb80897d5569f701b2418b8bc7f031ee.

Reply:
Thank you!